### DISCUSSION TITLE
bat GUI

### DISCUSSION THREAD

**Discussion by tshakalekholoane** [tshakalekholoane/bat#18] [Ideas]
I am thinking of experimenting with desktop GUI frameworks and part of that might include building a GUI for `bat` similar to this one running in Windows but for Linux.

<p align="center">
  <img src="https://user-images.githubusercontent.com/23554017/129551355-d842ccb9-06bd-4a5a-8215-8e8f0a0c7186.jpeg">
</p>

I realise that not everyone is familiar with command line interfaces and some features like [automatic updates](https://github.com/leveson/bat/discussions/17) might be easier to support in a GUI.

But enough people would have to be interested to make it worthwhile so if that's you, give this post a 👍 or 👎 so I can be able to gauge the level of interest. Also drop your email in the form linked [here](https://forms.gle/oEHh3jJj3QeEf7qT9) if you'd like to help out with testing whenever that comes out.

**Comment by sajidifti** [comment]
Great idea. If you can replicate the entire MyAsus app somehow, that will be even greater. Looking forward.

**Comment by GhosTHaise** [comment]
Is there a way to remove bat gui ?

**Reply by tshakalekholoane** [reply]
I am not sure I understand. There is no bat GUI.
